src/SklikApi.php

```diff
@@ -214,8 +218,14 @@ protected function requestAuthenticated(string $method, ?array $args = [], ?int
     protected function request(string $method, ?array $args = [], ?int $retries = self::RETRIES_COUNT): array
     {
         $decoder = new JsonDecode([JsonDecode::ASSOCIATIVE => true ]);
+        $retryPolicy = new SimpleRetryPolicy(5, [RequestException::class, ClientException::class]);
```

Comment:
Nemohla by to být `self::RETRIES_COUNT` konstanta, místo té `5`?

Reply:
jo, to by samozřejmě měly být konstanty :) díky